docs/s2i.md

```diff
@@ -0,0 +1,74 @@
+# s2i
```

Comment:
The page title should be using dashes, for instance:

```md
Source-to-Image Tasks (`s2i`)
-----------------------------
```

I also suggest a longer title as well, please.